openshift/e2e-common.sh

```diff
@@ -299,6 +299,9 @@ function run_e2e_tests(){
   sleep 30
   subdomain=$(oc get ingresses.config.openshift.io cluster  -o jsonpath="{.spec.domain}")
 
+  # Apply resource quota in rq-test namespace, needed for the related e2e test.
+  oc apply -f ./test/config/resource-quota/resource-quota.yaml
```

Comment:
Could you move this into `prepare_knative_serving_tests_nightly()`?
All `test/config/*` are applied there.

Reply:
I thought it could be added there but for example PVC yaml file was applied close to the PVC test.

Reply:
Ah, I missed it in https://github.com/openshift/knative-serving/pull/1076. Could you move both files into `prepare_knative_serving_tests_nightly()`?

Reply:
yes will do